Source/ZoomNet/Resources/IWebinars.cs

```diff
@@ -87,6 +106,26 @@ public interface IWebinars
 		/// </returns>
 		Task<Webinar> GetAsync(long webinarId, string occurrenceId = null, CancellationToken cancellationToken = default);
 
+		/// <summary>
+		/// Creates an existing recurring webinar for a user.
+		/// </summary>
+		/// <param name="webinarId">The webinar ID.</param>
+		/// <param name="topic">Webinar topic.</param>
+		/// <param name="agenda">Webinar description.</param>
+		/// <param name="start">Webinar start time.</param>
+		/// <param name="duration">Webinar duration (minutes).</param>
+		/// <param name="recurrence">Recurrence information.</param>
+		/// <param name="password">Password to join the webinar. By default the password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters. This can be updated within Zoom account settings.</param>
+		/// <param name="settings">Webinar settings.</param>
+		/// <param name="trackingFields">Tracking fields.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The async task.
+		/// Zoom will return an HTTP 200 response when there is no webinar subscription present.
```

Comment:
Are you kidding me!!?!?!?!?!? Zoom returns HTTP 200 (which is the universal code for 'success') when you don't have the necessary subscription?????? This is nuts!!!!

If that's the case, we should probably inspect the response and throw an exception to ensure developers are aware that the update failed.

Reply:
I thought it was a documentation error at first but have confirmed that it returns a 204 on our development account.

Reply:
Added handling to throw an error on a 200 response (hate that Zoom does this). Copied warning text directly from API documentation to avoid any confusion.

Couldn't find any other way to return `Task` while querying the response value to throw the error without using the `Response` object, not too keen on this approach, but it certainly does the job.

Reply:
To be honest, I was incredulous when I first read your comment about the HTTP 200 response but I reviewed the documentation and, sure enough, it explicitly says:

![image](https://user-images.githubusercontent.com/112710/105185157-34dca300-5afe-11eb-8d33-9b4ab6fed810.png)

I have interacted with a lot of API in my career and they all have their little quirks, but this one is definitely over the top!

Anyway, thanks for making this change. I prefer for our library to handle this situation and hide this "quirk" from developers.